<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StashDialog</class>
 <widget class="QDialog" name="StashDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>432</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>New stash</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="messageLabel">
     <property name="text">
      <string>&amp;Description</string>
     </property>
     <property name="buddy">
      <cstring>messageEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="messageEdit">
     <property name="placeholderText">
      <string>Optional stash message</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="optionsLabel">
     <property name="text">
      <string>Options</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="cleanupCheckBox">
     <property name="toolTip">
      <string>Normally, stashed changes are removed from the working directory.
Untick this to leave the stashed changes intact instead.</string>
     </property>
     <property name="text">
      <string>&amp;Remove stashed changes after stashing</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="willBackUpChangesLabel">
     <property name="text">
      <string>The stash will back up all changes to the files ticked below.</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="willRemoveChangesLabel">
     <property name="text">
      <string>These changes will then be removed from the working directory.</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="willKeepChangesLabel">
     <property name="text">
      <string>These changes will remain in the working directory.</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QListWidget" name="fileList">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="indexAndWtWarning">
     <property name="text">
      <string>Warning: Some of the files have both staged and unstaged changes. Those changes will be combined in the stash.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StashDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StashDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cleanupCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>willRemoveChangesLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cleanupCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>willKeepChangesLabel</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
